PSKoans/Koans/Foundations/AboutArrays.Koans.ps1

```diff
@@ -109,4 +183,77 @@ Describe 'Arrays' {
 
         # NOTE: The above will actually throw errors if you have PowerShell running in Strict Mode.
     }
+
+    It 'creates arrays from a range' {
+        # The .. notation used to reverse an array may be used to array.
+
+        $Array = 1, 2, 3, 4, 5
+
+        __..5 | Should -Be $Array
```

Comment:
I don't think this'll parse, except maybe as a nonexistent command name, which is not a very clear error to hand the user. You'll need to declare the variable first, I think.
```suggestion
        $StartIndex = __
        $StartIndex..5 | Should -Be $Array
```